daemons/controld/controld_callbacks.c

```diff
@@ -63,7 +63,7 @@ crmd_ha_msg_filter(xmlNode * msg)
         }
     }
 
-    /* crm_log_xml_trace(msg, "HA[inbound]"); */
+    // pcmk__log_xml_trace(msg, "HA[inbound]");
```

Comment:
This has been commented out at least as far back as 2012 (f61827edc22) so we can lose it too.